File: tests/core/resources/diff/go/function/diff.go








func fibonacci(n int) int {
	if n <= 1 {
		return 0
	}
	if n == 2 {
		return 1
	}

	prev, curr := 0, 1
	for i := 3; i <= n; i++ {
		prev, curr = curr, prev+curr
	}

	return curr
}

File: tests/core/resources/diff/npm/function/diff.js
function fibonacci(n) {
    if (n <= 1) return 0;
    if (n === 2) return 1;
  
    let prev = 0, curr = 1;
  
    for (let i = 3; i <= n; i++) {
      [prev, curr] = [curr, prev + curr];
    }
  
    return curr;
  }
